data/data/agent/systemd/units/assisted-service-pod.service

```diff
@@ -4,15 +4,19 @@ Wants=network.target node-zero.service
 After=network-online.target node-zero.service
 ConditionPathExists=/etc/assisted-service/node0
 RequiresMountsFor=
-Requires=assisted-service-db.service assisted-service.service assisted-service-ui.service
-Before=assisted-service-db.service assisted-service.service assisted-service-ui.service
+Requires=assisted-service-db.service assisted-service.service
+Before=assisted-service-db.service assisted-service.service
 
 [Service]
 Environment=PODMAN_SYSTEMD_UNIT=%n
 Restart=on-failure
 TimeoutStopSec=70
 ExecStartPre=/bin/rm -f %t/%n.pid %t/%N.pod-id
 ExecStartPre=/usr/bin/podman pod create --infra-conmon-pidfile %t/%n.pid --pod-id-file %t/%N.pod-id -n assisted-service --publish=8090:8090 --publish=8080:8080 --publish=8888:8888
+ExecStartPre=/usr/local/bin/release-image.sh %t agent-installer-node-agent
+ExecStartPre=/usr/local/bin/release-image.sh %t agent-installer-api-server
+ExecStartPre=/usr/local/bin/release-image.sh %t agent-installer-csr-approver
+ExecStartPre=/usr/local/bin/release-image.sh %t agent-installer-orchestrator
```

Comment:
Why not have the script just read all of the images at once and write them to the env file?

Reply:
Changed to do that - added new script get-container-images.sh